encrypted-media-respec.html

```diff
@@ -1240,7 +1259,31 @@
                         </dd>
                         <dt><a def-id="persistent-usage-record-session"></a></dt>
                         <dd>
-                          <p>Let <var>requested license type</var> be a non-persistable license that will persist a <var>record of key usage</var>.</p>
+                          <ol>
+                            <li>
+                              <p>Initialize this object's <var>record of key usage</var> as follows.</p>
+                              <ul>
+                                <li>
+                                  <p>
+                                    Set the list of <a def-id="key-id">key IDs</a> <a href="#known-key">known</a> to the session to the empty list.
+                                  </p>
+                                </li>
+                                <li>
+                                  <p>
+                                    Set the <var>first decrypt time</var> to <code>null</code>.
```

Comment:
Sounds good. As you say, this is internal state. (We might still change this later depending on the outcome of #59. Regardless, it doesn't impact the visible behavior...

Except, for Clear Key. Do we then need to handle the conversion from "null" to some value in the JSON? Would it be simpler to just use `NaN` here then?